### ISSUE TITLE
Why does gfs animation sometimes restart?

### ISSUE THREAD

**Issue by REAL-DoritosMAN** [luckydog7/Funkin-android#926]

**Comment by AnthonyCTM** [comment]
I have the same question

**Comment by REAL-DoritosMAN** [comment]
Same

**Comment by CreatorOlixter** [comment]
I think it's probably to sync with the bpm of the Song

**Comment by REAL-DoritosMAN** [comment]
> I think it's probably to sync with the bpm of the Song 

It makes sense but wouldn't that also happen on the title screen?

**Comment by CreatorOlixter** [comment]
> > I think it's probably to sync with the bpm of the Song
> 
> It makes sense but wouldn't that also happen on the title screen?

I Don't think so, cuz the bpm of the title Screen y some slow, like bopeebo. It noticed that happens on Fast songs

**Comment by REAL-DoritosMAN** [comment]
> > > I think it's probably to sync with the bpm of the Song
> > 
> > It makes sense but wouldn't that also happen on the title screen?
> 
> I Don't think so, cuz the bpm of the title Screen it's some slow, like bopeebo. It noticed that happens on Fast songs

I changed the title screen music and the animation plays faster but doesn't restart
